simulator/src/main/java/Simulator.java

```diff
@@ -49,12 +47,13 @@ private void sleep(int ms)
 
     private void constructDevices()
     {
-        int numDevices = (Integer) this.args.get(Arg.NUMDEVICES);
-        int[] signalStrenths = (int[]) this.args.get(Arg.SIGNAL_STRENGTHS);
+//        int numDevices = (Integer) this.args.get(Arg.NUMDEVICES);
+//        int[] signalStrenths = (int[]) this.args.get(Arg.SIGNAL_STRENGTHS);
```

Comment:
Should this commented out code be removed? Is it being used?

Reply:
Once we get the router working on the devices, im going to get a better config working for the simulator (cmd args or config file) so these can stay for now but will be removed when a better solution is developed